### PR TITLE
Fixes badges for Ruby

### PR DESCRIPTION
**Pull request by gguuss**
Setting as draft until blocking PRs are merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Ruby docs samples
 
-Kokoro: [![Kokoro](https://storage.googleapis.com/cloud-devrel-kokoro-resources/ruby/ruby-docs-samples/system_tests-ubuntu.png)](https://fusion.corp.google.com/projectanalysis/current/KOKORO/prod:cloud-devrel%2Fruby%2Fruby-docs-samples%2Fsystem_tests)
```
Comment (tbpg) [inline README.md:3]: I think the intention is for this to be the AND of the other two jobs? But, sometimes the permissions get messed up on the GCS object -- I'm not sure why.

Comment (gguuss) [reply README.md:3]: Last week, I updated the kokoro build to separate the results for each and generate separate HTML output.

**Comment by gguuss** [comment]
Should render as:

Ruby 2.4: [Kokoro](https://storage.googleapis.com/cloud-devrel-public/ruby-docs-samples/badges/ruby2.4.html)
Ruby 2.5: [Kokoro](https://storage.googleapis.com/cloud-devrel-public/ruby-docs-samples/badges/ruby2.5.html)

File: README.md (hunk 1 above)
Comment (gguuss) [reply README.md:3]: I believe the older configuration was failing because the badge was being generated within the presubmit.

**Pull request merged**